### DISCUSSION TITLE
[Aqara water leak] Problema status dispositivo

### DISCUSSION THREAD

**Discussion by baldus88** [indomus/forum#2298] [ZigBee, Z-Wave, Thread, Matter]
Ciao.
Ho da poco ripristinato la mia rete zigbee2mqtt tramite docker e mi funzionano tutti i dispositivi a parte uno, che forse è il caso magari aveva già altri problemi di suo, ma non mi vede più i valori, me li da null:

![image](https://user-images.githubusercontent.com/57867040/213869578-37bde637-ae21-409f-89e6-1249940ab299.png)

Ho provato pure a rimouovere e rimetterlo ma niente.
Dai log in debug mi da questo:

```
Debug 2023-01-21 14:40:56Received Zigbee message from 'acqua_bagno', type 'attributeReport', cluster 'genBasic', data '{"65281":{"1":3015,"10":0,"100":1,"3":21,"4":5032,"5":116,"6":[0,0],"8":516},"modelId":"lumi.sensor_wleak.aq1"}' from endpoint 1 with groupID 0
Debug 2023-01-21 14:40:56lumi.sensor_wleak.aq1: unknown key 6 with value 0,0
Debug 2023-01-21 14:40:56lumi.sensor_wleak.aq1: Processed data into payload {"voltage":3015,"battery":100,"device_temperature":21,"power_outage_count":115}
Info 2023-01-21 14:40:56MQTT publish: topic 'zigbee2mqtt/acqua_bagno', payload '{"battery":100,"battery_low":null,"device":{"applicationVersion":4,"dateCode":"20170721","friendlyName":"acqua_bagno","hardwareVersion":2,"ieeeAddr":"0x00158d00042b16b5","manufacturerID":4151,"manufacturerName":"LUMI","model":"SJCGQ11LM","networkAddress":1072,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"device_temperature":21,"linkquality":31,"power_outage_count":115,"voltage":3015,"water_leak":null}'
```

Qualcuno può darmi una mano e capire da cosa può essere? L'unica cosa che mi è venuta in mente è quel "unknown key 6" ma cercando in giro non ho trovato molto.

Grazie!

**Comment by snakuzzo** [comment]
Sinceramente, non saprei.
C'è solo una issue relativa ad un problema simile [QUI](https://github.com/Koenkk/zigbee2mqtt/issues/14650), ma senza alcun dettaglio.
Hai ovviamente provato a bagnarlo per vedere se cambia stato, giusto ?

**Reply by baldus88** [reply]
Tra le prove che avevo fatto ero convinto di averlo bagnato ma misà no l'ultima volta che l'ho rimesso... sta di fatto che ora funziona :)

Le altre volte appena lo mettevo ero convinto che già prendesse in "dry" senza fare nulla ma forse ricordo male.

Grazie ancora del supporto.
